packages/amplify-graphql-model-transformer/rds-lambda/handler.ts

```diff
@@ -18,19 +20,30 @@ const createSSMClient = (): void => {
   secretsClient = new SSMClient({});
 };
 
-const getSSMValue = async(key: string | undefined): Promise<string> => {
+const wait10SecondsAndThrowError = async (): Promise<void> => {
+  await delay(10000);
+  console.log('Unable to retrieve secret for database connection from SSM. If your database is in VPC, verify that you have VPC endpoints for SSM defined and the security group\'s inbound rule for port 443 is defined.');
```

Comment:
Instead of console.log, should probably add a log to the transformer logger methods https://github.com/aws-amplify/amplify-category-api/blob/main/packages/amplify-graphql-transformer-core/src/transformation/transformer-plugin-base.ts#L60-L62

Reply:
This logic resides in the SQL lambda deployed on the customer account.